Fix segfault when using (hot)key on bond twice in a row. Taken from
https://github.com/bryanherger/xdrawchem/pull/12

Index: xdrawchem/render2d_text.cpp
--- xdrawchem/render2d_text.cpp.orig
+++ xdrawchem/render2d_text.cpp
@@ -380,6 +380,9 @@ void Render2D::keyPressEvent( QKeyEvent * k )
                 }
 
                 if ( highlightobject->Type() != TYPE_TEXT ) {
+                    delete localtexteditor;
+                    localtexteditor = 0;
+
                     hotkeymode = false;
                     mode = hkprevmode;
                     return;
